<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1251</width>
    <height>741</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DataLogger</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>0</height>
           </size>
          </property>
          <property name="title">
           <string>Readouts</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QFrame" name="frame_4">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QGridLayout" name="gridLayout_3">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item row="1" column="0" rowspan="2">
               <widget class="QLCDNumber" name="geigerLCD">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                  <horstretch>2</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="autoFillBackground">
                 <bool>true</bool>
                </property>
                <property name="segmentStyle">
                 <enum>QLCDNumber::Flat</enum>
                </property>
               </widget>
              </item>
              <item row="1" column="1" alignment="Qt::AlignRight">
               <widget class="QLabel" name="geigerIndicator">
                <property name="minimumSize">
                 <size>
                  <width>16</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>16</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">
    border: 0px solid #555;
    border-radius: 8px;
    border-style: outset;
	background-color: rgb(255, 0, 0);
    padding: 5px;</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="0" column="0" colspan="2">
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>Geiger Counter</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QComboBox" name="geigerUnits">
                <property name="minimumSize">
                 <size>
                  <width>60</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="editable">
                 <bool>false</bool>
                </property>
                <item>
                 <property name="text">
                  <string>CPM</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>CPS</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>mR/hr</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>μSv/hr</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame_2">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QGridLayout" name="gridLayout_2">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item row="1" column="0" rowspan="2">
               <widget class="QLCDNumber" name="gaussLCD">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                  <horstretch>2</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="autoFillBackground">
                 <bool>true</bool>
                </property>
                <property name="segmentStyle">
                 <enum>QLCDNumber::Flat</enum>
                </property>
               </widget>
              </item>
              <item row="1" column="1" alignment="Qt::AlignRight">
               <widget class="QLabel" name="gaussIndicator">
                <property name="minimumSize">
                 <size>
                  <width>16</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>16</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">
    border: 0px solid #555;
    border-radius: 8px;
    border-style: outset;
	background-color: rgb(255, 0, 0);
    padding: 5px;</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="2" column="1" alignment="Qt::AlignRight">
               <widget class="QComboBox" name="gaussUnits">
                <property name="minimumSize">
                 <size>
                  <width>60</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="editable">
                 <bool>false</bool>
                </property>
                <item>
                 <property name="text">
                  <string>mG</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>μT</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="0" column="0" colspan="2">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Gauss Meter</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame_5">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QGridLayout" name="gridLayout_4">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item row="0" column="0" colspan="2">
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>RF Meter</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1" alignment="Qt::AlignRight">
               <widget class="QComboBox" name="rfUnits">
                <property name="minimumSize">
                 <size>
                  <width>60</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>60</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="editable">
                 <bool>false</bool>
                </property>
                <item>
                 <property name="text">
                  <string>μW/m²</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>mW/m²</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="1" column="1" alignment="Qt::AlignRight">
               <widget class="QLabel" name="rfIndicator">
                <property name="minimumSize">
                 <size>
                  <width>16</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>16</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">
    border: 0px solid #555;
    border-radius: 8px;
    border-style: outset;
	background-color: rgb(255, 0, 0);
    padding: 5px;</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="1" column="0" rowspan="2">
               <widget class="QLCDNumber" name="rfLCD">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Ignored" vsizetype="Minimum">
                  <horstretch>2</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="autoFillBackground">
                 <bool>true</bool>
                </property>
                <property name="segmentStyle">
                 <enum>QLCDNumber::Flat</enum>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="spectrumButton">
             <property name="text">
              <string>Open Spectrum Analyzer...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="recButton">
             <property name="text">
              <string>Record</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="filenameLabel">
             <property name="text">
              <string>Not currently logging to file.</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="filenameEdit">
             <property name="placeholderText">
              <string>Text to prepend to filename. Enter before recording.</string>
             </property>
            </widget>
           </item>
          </layout>
          <zorder>frame_2</zorder>
          <zorder>recButton</zorder>
          <zorder>spectrumButton</zorder>
          <zorder>frame_4</zorder>
          <zorder>frame_5</zorder>
          <zorder>filenameLabel</zorder>
          <zorder>filenameEdit</zorder>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QGroupBox" name="groupBox_4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Options</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Geiger Counter Mode</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="timeAvgMode">
               <property name="text">
                <string>Time Averaged</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">geigerModeButtonGroup</string>
               </attribute>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rawCountMode">
               <property name="text">
                <string>Raw Counts</string>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">geigerModeButtonGroup</string>
               </attribute>
              </widget>
             </item>
             <item>
              <widget class="Line" name="line">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>RF Meter Ext. Adapter</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rf0db">
               <property name="text">
                <string>0 dB</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">rfAdapterButtonGroup</string>
               </attribute>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rfMinus20db">
               <property name="text">
                <string>-20 dB</string>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">rfAdapterButtonGroup</string>
               </attribute>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rfPlus20dB">
               <property name="text">
                <string>+20dB</string>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">rfAdapterButtonGroup</string>
               </attribute>
              </widget>
             </item>
             <item>
              <widget class="Line" name="line_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>RF Meter Range</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rfRangeMax">
               <property name="text">
                <string>Max [20000 μW/m²]</string>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">rfRange</string>
               </attribute>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rfRangeMin">
               <property name="text">
                <string>Min [2000 μW/m²]</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">rfRange</string>
               </attribute>
              </widget>
             </item>
             <item>
              <widget class="Line" name="line_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Max Samples Per File</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="maxSampleSelector">
               <item>
                <property name="text">
                 <string>3600</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>7200</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>14400</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>21600</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>43200</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Sample Interval [s]</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="intervalSelector">
               <item>
                <property name="text">
                 <string>1</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>2</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>5</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>10</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>15</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>30</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Log</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QTableView" name="logTable">
             <property name="palette">
              <palette>
               <active>
                <colorrole role="Base">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>240</red>
                   <green>240</green>
                   <blue>240</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="AlternateBase">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>208</red>
                   <green>208</green>
                   <blue>208</blue>
                  </color>
                 </brush>
                </colorrole>
               </active>
               <inactive>
                <colorrole role="Base">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>240</red>
                   <green>240</green>
                   <blue>240</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="AlternateBase">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>208</red>
                   <green>208</green>
                   <blue>208</blue>
                  </color>
                 </brush>
                </colorrole>
               </inactive>
               <disabled>
                <colorrole role="Base">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>255</red>
                   <green>0</green>
                   <blue>0</blue>
                  </color>
                 </brush>
                </colorrole>
                <colorrole role="AlternateBase">
                 <brush brushstyle="SolidPattern">
                  <color alpha="255">
                   <red>208</red>
                   <green>208</green>
                   <blue>208</blue>
                  </color>
                 </brush>
                </colorrole>
               </disabled>
              </palette>
             </property>
             <property name="styleSheet">
              <string notr="true">gridline-color: rgba(255, 255, 255, 0)</string>
             </property>
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>2</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Graphs</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="GraphWidget" name="graph" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QMenu" name="menuGeiger">
     <property name="title">
      <string>Geiger Counter</string>
     </property>
     <addaction name="actionAveraging_Time"/>
     <addaction name="separator"/>
     <addaction name="avg10s"/>
     <addaction name="avg30s"/>
     <addaction name="avg60s"/>
    </widget>
    <addaction name="menuGeiger"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCPM">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>CPM</string>
   </property>
  </action>
  <action name="actionCPM_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>CPM</string>
   </property>
  </action>
  <action name="actionCPS">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>CPS</string>
   </property>
  </action>
  <action name="actionmR_Hr">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>mR/hr</string>
   </property>
  </action>
  <action name="actionuSv_hr">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>uSv/hr</string>
   </property>
  </action>
  <action name="actionmG">
   <property name="text">
    <string>mG</string>
   </property>
  </action>
  <action name="actionuT">
   <property name="text">
    <string>uT</string>
   </property>
  </action>
  <action name="actionHello">
   <property name="text">
    <string>Hello</string>
   </property>
  </action>
  <action name="actionAveraging_Time">
   <property name="text">
    <string>Averaging Time</string>
   </property>
  </action>
  <actiongroup name="avgTimeMenu">
   <action name="avg10s">
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>10s</string>
    </property>
   </action>
   <action name="avg30s">
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>30s</string>
    </property>
   </action>
   <action name="avg60s">
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>60s</string>
    </property>
   </action>
  </actiongroup>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphWidget</class>
   <extends>QWidget</extends>
   <header>qtgraphwidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="geigerModeButtonGroup"/>
  <buttongroup name="rfAdapterButtonGroup"/>
  <buttongroup name="rfRange"/>
 </buttongroups>
</ui>
